storage/skip_list/src/skip_list.rs

```diff
@@ -157,6 +191,56 @@ impl<C: ListValue> SkipList<C> {
             f64::log((index + 1) as f64, LEVEL_BASE as f64).floor() as usize
         }
     }
+
+    /// Build a difference object between two indexes.
+    /// `list.get(from).merge(list.diff(from, to)) == list.get(to)`
+    #[allow(dead_code)]
```

Comment:
don't forget ot remove

Reply:
We will surely get to back to this, I would recommend to merge this, as it shows at least base implementation, of what we are trying to do.